### PR TITLE
Update DG, add ppp

### PR DESCRIPTION
**Pull request by amitrahman1026**

### CODE REVIEW

File: src/test/java/seedu/duke/utils/StateTest.java [hunk 1]
```diff
@@ -16,22 +16,8 @@ public class StateTest {
     public void testGetSelectedModulesList_correctSelectedModuleList() {
         // validate correct module details
         Module module1 = Module.get("CS1010S");
-        assertNotNull(module1);
-        assertEquals("CS1010S", module1.moduleCode);
```
Comment (owenl131) [inline src/test/java/seedu/duke/utils/StateTest.java:20]: Honestly this test class shouldn't be testing for these

File: src/test/java/seedu/duke/utils/StateTest.java [hunk 2]
```diff
@@ -169,4 +126,22 @@ void testRemoveSelectedModule_returnedListEqualsGenericListOfSelectedModulesAdde
         assertEquals(returnedListOfAddedModules, list);
     }
 
+    void validateTwoTestModulesCS1010SandCS1231(Module module1, Module module2) {
```
Comment (owenl131) [inline src/test/java/seedu/duke/utils/StateTest.java:129]: Cos these details are for the Module class to test.

**Review by owenl131 — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/duke/utils/StateTest.java (hunk 1 above)
Comment (amitrahman1026) [reply src/test/java/seedu/duke/utils/StateTest.java:20]: oops, missed that out thanks

**Review by kohnh — APPROVED** [review]
LGTM

**Pull request merged**